tests/robustness/traffic/key_window.go

```diff
@@ -0,0 +1,62 @@
+package traffic
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+)
+
+type KeyWindow struct {
+	sync.Mutex
+
+	Counter    int
+	WindowSize int
+	Keys       []int
```

Comment:
Maybe Key field should include key names and not numbers used to generated keys.